definitions/3.0.0/messageExampleObject.json

```diff
@@ -24,9 +24,10 @@
     },
     "headers": {
       "type": "object",
-      "description": "Example of the application headers. It can be of any type."
+      "description": "Example of the application headers. MUST be a map of key-value pairs."
     },
     "payload": {
+      "type": ["number", "string", "boolean", "object", "array", "null"],
```

Comment:
Why maintaining a list of types here? 🤔

Reply:
In previous versions here was `any` type. I found in SO that's better way to simulate any type and don't broke validation is to enumerate each JSON Schema type

Reply:
Why is it going to break validation? When `type` is not specified, it can be any type, right? If some tools don't get it right, shouldn't it be fixed in the tools instead?

Reply:
It's either a string or an array of strings but it's not required, so we can decide not to put it there. Effectively, I think it means "any" but good idea pinging the JSON Schema folks as, in the end, they know best.

Reply:
Anyway, I'm ready to remove `type` and patch our schema for JetBrains IDE and other validators, as I have already done, previously

My main concern is to try to bring changes back, here, to make our schema compatible with various IDEs and environments where AsyncAPI specification will be validated via our JSON Schemas

And be sure that this changes are synced with JSON Schema specification requirements

Reply:
Understandable mate. It's a fine line between trying to be pragmatic and keeping the spec strict. I'm not against maintaining this list of types but code you don't write is code you don't have to maintain. Arguably, explicit is better than implicit. Let's see what others have to say 🤷‍♂️ I'm not going to block this PR because of this line.